constants/labels.js

```diff
@@ -0,0 +1,64 @@
+
+const prefixes = {
+    CUSTOMER: 'Customer',
+    BUG: 'Bug',
+    FEATURE: 'Feature'
+};
+
+const bugTypes = {
+    PRODUCTION: 'production',
+    DEVELOPMENT: 'development'
+}
+
+const featureTypes = {
+    NEW: 'new',
+    EXISTING: 'existing'
+}
+
+const getPrefix = (label) => (label || '').split(':')[0].trimRight();
+
+function getType (label) {
+    const labelType = (label || '').split(':');
+    return (labelType.length > 1) ? labelType[1].trimLeft() : '';
+}
```

Comment:
Think these two functions can be combined:

```suggestion
const parseLabel = label => {
  if (!label) return { group: null, type: null };
  const [group = '', type= ''] = label.split(';');
  return {
    group: group.trimRight(),
    type: type.trimLeft(),
  };
}
```